### ISSUE TITLE
Handle livestream-fetch delay in Following section

### ISSUE THREAD

**Issue by infinite-persistence** [OdyseeTeam/odysee-frontend#1624]
We've went back-and-forth between waiting and not waiting for livestreams in the Following Section in Homepage.  Currently, we are back to waiting for it to support the even-grid changes, but now I often get a blank (albeit with placeholders) Following Section because the API is slow in my region.

"tile evenness" vs. "quick display" -- the latter feels more important ... and also for Core Vitals LCP consideration.

----

<img width="400" alt="image" src="https://user-images.githubusercontent.com/64950861/171565325-cad2aa38-1065-4d87-8c9e-90616207407f.png">

<img width="400" alt="image" src="https://user-images.githubusercontent.com/64950861/171565755-a49c0757-241d-4de7-9434-415639634f7b.png">

**Comment by tzarebczan** [comment]
@nikooo777 can we try putting this one behind Cloudflare to see if it would help?

**Comment by tzarebczan** [comment]
And I'm okay with going back to non waiting if it's feasible and not too difficult.

**Comment by nikooo777** [comment]
yeah this is definitely stupid. It looks like we were not gzipping responses from the API so I added that. can you check and see if it's any faster/better?
I see a big improvement on my end already

**Comment by infinite-persistence** [comment]
Yay, the call reduced to ~500ms.

Next is an app-side problem:
It still needs to make a series of paginated `claim_search` (to resolve them?), so that 2-3s wasted.

<img width="439" alt="image" src="https://user-images.githubusercontent.com/64950861/172515140-18a9a4c8-24b3-4db0-bfd4-c3e55cf1a293.png">

I say we ignore the layout shift for this case (lesser of 2 evils), because it is shifting anyways sometimes:

https://user-images.githubusercontent.com/64950861/172516500-38179577-a745-46e3-8488-5946b54e7cf0.mp4

**Comment by tzarebczan** [comment]
We still do have random spikes due to other issues which affect response times...not often, but does happen.

Regarding the claim searches - we can use the new API data to minimize them, but it doesn't have future stream support yet. If there's over 50, we'll Still have to paginate claim searches, but can be improved to parallelize since we have predefined counts of live (and later scheduled) streams.

**Comment by infinite-persistence** [comment]
> _I say we ignore the layout shift for this case (lesser of 2 evils), because it is shifting anyways sometimes:_

On second thought, having a shift within 1s after load is better than shifting 3-4s after load, since the user could be doing something by then.

> _can be improved to parallelize_

Good idea. Didn't thought of it, as I assumed it needs to be called in sequence to avoid results jumping from one page to another.
